cmd/uplink/cmd/meta_get.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"storj.io/storj/internal/fpath"
+	"storj.io/storj/pkg/process"
+)
+
+func init() {
+	addCmd(&cobra.Command{
+		Use:   "get [KEY] PATH",
+		Short: "Get a Storj object's metadata",
+		RunE:  metaGetMain,
+	}, metaCmd)
+}
+
+// metaGetMain is the function executed when metaGetCmd is called.
+func metaGetMain(cmd *cobra.Command, args []string) (err error) {
+	var key *string
+	var path string
+
+	switch len(args) {
+	case 0:
+		return fmt.Errorf("No object specified")
+	case 1:
+		path = args[0]
+	case 2:
+		key = &args[0]
+		path = args[1]
+	default:
+		return fmt.Errorf("Too many arguments")
+	}
+
+	ctx, _ := process.Ctx(cmd)
+
+	src, err := fpath.New(path)
+	if err != nil {
+		return err
+	}
+
+	project, bucket, err := cfg.GetProjectAndBucket(ctx, src.Bucket())
+	if err != nil {
+		return err
+	}
+	defer closeProjectAndBucket(project, bucket)
+
+	object, err := bucket.OpenObject(ctx, src.Path())
+	if err != nil {
+		return err
+	}
+
+	if key != nil {
+		value, ok := object.Meta.Metadata[*key]
+		if ok != true {
+			return fmt.Errorf("Key does not exist")
```

Comment:
If key doesn't exists then I have large stacktrace
```
2019-10-22T10:46:12.542+0200	FATAL	process/exec_conf.go:299	Unrecoverable error	{"error": "Key does not exist"}
storj.io/storj/pkg/process.cleanup.func1
	/home/wywrzal/git/storj/pkg/process/exec_conf.go:299
github.com/spf13/cobra.(*Command).execute
	/home/wywrzal/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826
github.com/spf13/cobra.(*Command).ExecuteC
	/home/wywrzal/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
github.com/spf13/cobra.(*Command).Execute
	/home/wywrzal/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
storj.io/storj/pkg/process.Exec
	/home/wywrzal/git/storj/pkg/process/exec_conf.go:74
main.main
	/home/wywrzal/git/storj/cmd/uplink/main.go:12
runtime.main
	/usr/local/go/src/runtime/proc.go:203
```
Maybe we can just print message as a result and not return error in this case?

Reply:
This is also the case with number of parameters check. Instead of simple message like `No object specified` we have large stacktrace.

Reply:
https://github.com/storj/storj/pull/3310#discussion_r336653160

I believe the traces only happen on non-release builds.

Reply:
yes, you are right I forgot about it